### PR TITLE
DDF-3915 Solr documentation updates

### PR DESCRIPTION
**Pull request by ahoffer**
##### ABBREVIATED REVIEW BETWEEN 2.13.X AND MASTER IS IN EFFECT
Link to 2.13.x PR: https://github.com/codice/ddf/pull/3518

#### What does this PR do?
Updates documentation related to standalone Solr server epic

#### Who is reviewing it? 
@mcalcote 
@austinsteffes 
@shaundmorris 
@emmberk 

#### Ask 2 committers to review/merge the PR and tag them here.
@figliold
@ricklarsen - Documentation
@rzwiefel

#### How should this be tested?
Checkout the branch and perform a clean build of the distribution/docs module. 
Inspect the output, `target/docs/html/documentation.html` for proper formatting, spelling, grammar.

#### Checklist:
- [X] Documentation Updated
- [NA] Update / Add Unit Tests
- [NA] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/docs/src/main/resources/content/_configuring/solr-admin-ui.adoc [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+If the browser is not correctly configured with a
+certificate that Solr trusts, the browser displays an error message about client authentication
+failing, or a message that the client certificate is invalid.
+
+=== Solr Admin UI's URL
+
+The Solr server's URL is configured in ${branding}'s `system.properties` file. See
+<<{managing-prefix}solr_server_url,solr.http.url>> for more information.
+An _example_ of a typical URL for the Solr Admin UI is `https//:hostname:8994`.
```
Comment (mcalcote) [inline distribution/docs/src/main/resources/content/_configuring/solr-admin-ui.adoc]: ✏️ `https://`

**Comment by ricklarsen** [comment]
build now

**Comment by ahoffer** [comment]
I did not inspect the complete Jenkins log, but I do think the documentation updates caused this CI build to fail. I would like to this PR merged despite the fact that it has no successful CI build.

```17:00:10 [INFO] ------------------------------------------------------------------------
17:00:10 [INFO] Reactor Summary:
17:00:10 [JENKINS] Archiving disabled[INFO] 
17:00:10 [INFO] DDF :: UI .......................................... SUCCESS [04:51 min]
17:00:10 [INFO] DDF DOCS ........................................... SUCCESS [01:50 min]
17:00:10 [INFO] DDF :: Distribution :: DDF ......................... SUCCESS [02:26 min]
17:00:10 [INFO] DDF :: Integration :: Test ......................... SUCCESS [35:51 min]
17:00:10 [INFO] Pom Fix ............................................ SUCCESS [ 10.770 s]
17:00:10 [INFO] Pom Fix Run ........................................ SUCCESS [  8.078 s]
17:00:10 [INFO] ------------------------------------------------------------------------
17:00:10 [INFO] BUILD SUCCESS
17:00:10 [INFO] ------------------------------------------------------------------------
17:00:10 [INFO] Total time: 53:47 min
17:00:10 [INFO] Finished at: 2018-08-02T17:00:10+00:00
17:00:10 
17:00:13 [INFO] Final Memory: 566M/1353M
17:00:13 [INFO] ------------------------------------------------------------------------
17:00:23 channel stopped
17:00:23 using settings config with name phxSettings
17:00:23 Replacing all maven server entries not found in credentials list is true
17:00:23 using global settings config with name phxOfficeGlobalSettings
17:00:23 Replacing all maven server entries not found in credentials list is true
17:00:23 [Linux] $ /jenkins/tools/hudson.tasks.Maven_MavenInstallation/Maven_3.3.9/bin/mvn -s /tmp/settings6301730580401450909.xml -gs /tmp/global-settings1024922229539423234.xml -fn -q -B -Dsonar.github.pullRequest=3537 -Dsonar.github.oauth=**** sonar:sonar -Dsonar.analysis.mode=preview -Dsonar.host.url=https://sonarcloud.io -Dsonar.login=**** -Dsonar.organization=codice -Dsonar.projectKey=ddf -pl !/distribution/docs,!distribution/test/itests/test-itests-ddf -Dsonar.github.repository=codice/ddf -Dnode.download.root=http://nexus3.phx.connexta.com/repository/nodejs.org/dist/ -Dyarn.download.root=http://nexus3.phx.connexta.com/repository/yarnpgk/ -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
17:10:47 FATAL: command execution failed
17:10:47 java.nio.channels.ClosedChannelException
17:10:47 	at org.jenkinsci.remoting.protocol.NetworkLayer.onRecvClosed(NetworkLayer.java:154)
17:10:47 	at org.jenkinsci.remoting.protocol.impl.NIONetworkLayer.ready(NIONetworkLayer.java:179)
17:10:47 	at org.jenkinsci.remoting.protocol.IOHub$OnReady.run(IOHub.java:721)
17:10:47 	at jenkins.util.ContextResettingExecutorService$1.run(ContextResettingExecutorService.java:28)
17:10:47 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
17:10:47 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
17:10:47 	at java.lang.Thread.run(Thread.java:748)
17:10:47 Caused: java.io.IOException: Backing channel 'JNLP4-connect connection from himym.phx.connexta.com/10.101.190.105:32866' is disconnected.```

**Comment by rzwiefel** [comment]
Manual hero successful. Built and docs look good locally.

**Comment by ricklarsen** [comment]
build now

**Comment by ricklarsen** [comment]
merging per manual hero verification

**Pull request merged**